.github/terraform/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "branch" {
+  description = "Github Action Branch Name. This will set Observe Datastream Token Name"
```

Comment:
missing type - linter usually bitches about this

Reply:
Added